examples/maps_heatmaps/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "module": "esnext",
+    "module": "CommonJS",
```

Comment:
@TalTamir86 Can you reply with an explaination/URL to my comment on why this change was necessary?

Reply:
https://github.com/TypeStrong/ts-node/issues/922
https://stackoverflow.com/questions/58384179/syntaxerror-cannot-use-import-statement-outside-a-module (in option 2)